### PR TITLE
[15.0][FIX] product_supplierinfo_for_customer_elaboration: Wrong for multiple elaborations. Tests was broken.

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

ping @CarlosRoca13 @ernestotejeda

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by sergio-teruel** [comment]
@ernestotejeda

**Review by ernestotejeda — APPROVED** [review]
Code and Functional review 👍🏼

**Review by ernestotejeda — APPROVED** [review]
Code and Functional review 👍🏼

File: product_supplierinfo_for_customer_elaboration/models/sale_order_line.py [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
+        for line in self:
+            customer_info = line._get_product_customer_info()
+            if customer_info:
+                line.elaboration_ids = customer_info.elaboration_ids
+
+    def _compute_elaboration_note(self):
+        res = super()._compute_elaboration_note()
+        for line in self:
+            customer_info = line._get_product_customer_info()
+            if customer_info and line.elaboration_ids:
```
Comment (pedrobaeza) [inline product_supplierinfo_for_customer_elaboration/models/sale_order_line.py]:
```suggestion
            if customer_info.elaboration_ids:
```

File: product_supplierinfo_for_customer_elaboration/models/sale_order_line.py [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
+from odoo import api, fields, models
 
 
 class SaleOrderLine(models.Model):
     _inherit = "sale.order.line"
 
-    @api.onchange("product_id")
-    def product_id_change(self):
-        res = super().product_id_change()
+    elaboration_ids = fields.Many2many(
```
Comment (pedrobaeza) [inline product_supplierinfo_for_customer_elaboration/models/sale_order_line.py:11]: Why do you need this field? It seems useless with the proposed change.

**Pull request merged**